filebeat/module/osquery/result/_meta/fields.yml

```diff
@@ -0,0 +1,27 @@
+- name: result
+  type: group
+  description: >
+    Common fields exported by the result metricset.
+  fields:
+    - name: name
+      type: keyword
+      description: >
+        The name of the query that generated this event.
+    - name: action
+      type: keyword
+      description: >
+        For incremental data, marks whether the entry was added
+        or removed. It can be one of "added", "removed", or "snapshot".
+    - name: host_identifier
+      type: keyword
+      description: >
+        The identifier for the host on which the osquery agent is running.
+        Normally the hostname.
+    - name: unix_time
+      type: long
+      description: >
+        Unix timestamp of the event. Used for computing the `@timestamp` column.
+    - name: calendar_time
+      tupe: keyword
+      description: >
+        String representation of the collection time, as formatted by osquery.
```

Comment:
Interesting that this is called `calendar_time`. I like the description with `collection time`. I wonder if we should have a common field in the future for the `collection_time` (or a different name).

Reply:
In osquery, they have `unixTime` and `calendarTime` which are different representations of the same moment. These get translated to `unix_time` and `calendar_time` by the module, and `@timestamp` is computed from `unix_time`. So there's quite a bit of redundancy, we could consider dropping the `calendar_time` by default.

Reply:
I added `read_time` (btw, it's not prefixed by `beat`) as being the read time when filebeat reads the log lines.